c3/generator/devices.py

```diff
@@ -182,6 +185,53 @@ def process(self, instr, chan, mixed_signal):
         return self.signal
 
 
+@dev_reg_deco
+class Crosstalk(Device):
+    """
+    Device to phenomenologically include crosstalk in the model by explicitly mixing
+    drive lines.
+
+    Parameters
+    ----------
+
+    crosstalk_matrix: tf.constant
+
+    """
+
+    def __init__(self, **props):
+        self.crossed_channels = props.pop("channels", None)
+        super().__init__(**props)
+        self.inputs = props.pop("inputs", 1)
+        self.outputs = props.pop("outputs", 1)
+        self.params["crosstalk_matrix"] = props.pop("crosstalk_matrix", None)
+
+    def process(self, signal: dict) -> dict:
+        """
+        Mix channels in the input signal according to a crosstalk matrix.
+
+        Parameters
+        ----------
+        instr: Instruction
```

Comment:
Can you add examples for what `instr`, `chan` and `signal` should look like (right inside the docstrings using an additional Examples header). Similarly what does the expected return output dict look like and what is the intended use of this module?

Ref: https://numpydoc.readthedocs.io/en/latest/format.html - point 13

Reply:
done